macros/utils/cross_db_utils/generate_surrogate_key.sql

```diff
@@ -0,0 +1,8 @@
+{% macro generate_surrogate_key(field_list) -%}
```

Comment:
I tried It. 
@IDoneShaveIt 
https://github.com/elementary-data/dbt-data-reliability/pull/174#discussion_r1040779731